### DISCUSSION TITLE
Joint project using MIMIC

### DISCUSSION THREAD

**Discussion by nikolosof** [MIT-LCP/mimic-code#1632] [MIMIC-General]
Greetings!

I have a question about MIMIC access and working on a project with MIMIC as a team.

I am a credentialed user and I have an idea for a project using MIMIC that I have shared with my supervisor at my institution. Do they have to be a credentialed user for me to show him the results of my sql query? 
I plan to show them the final table/results and would not show them any other parts of MIMIC. 

There is already a discussion on working jointly on MIMIC data, however in my situation only I will be working with the database directly. I haven't found an answer for this scenario. 

Thank you!

**Comment by alistairewj** [comment]
Yes, if they would like to look at MIMIC data they should be credentialed.
